### ISSUE TITLE
[travis-tasks] Source IPs for job notification webhooks do not match documentation

### ISSUE THREAD

**Issue by soulshake** [travis-ci/travis-ci#9158]
Job-related notifications should originate from these addresses ([source](https://docs.travis-ci.com/user/notifications/#Note-on-IP-addresses)):

- `54.173.229.200`
- `54.175.230.252`

Instead, they are coming from a variety of other IPs, including:

```
54.161.125.148 [26/Jan/2018:07:06:30.483]
54.161.125.148 [26/Jan/2018:07:07:06.758]
54.167.233.250 [26/Jan/2018:07:35:28.844]
54.167.233.250 [26/Jan/2018:07:48:08.812]
54.81.204.180 [26/Jan/2018:07:50:24.415]
54.167.233.250 [26/Jan/2018:08:12:33.893]
54.227.57.109 [26/Jan/2018:08:12:55.247]
54.166.55.153 [26/Jan/2018:09:11:37.066]
54.161.125.148 [26/Jan/2018:09:14:08.576]
54.161.125.148 [26/Jan/2018:09:39:31.489]
54.166.55.153 [26/Jan/2018:09:43:08.225]
54.161.125.148 [26/Jan/2018:09:43:13.919]
54.167.3.137 [26/Jan/2018:09:48:09.527]
54.161.125.148 [26/Jan/2018:09:54:13.899]
54.81.197.243 [26/Jan/2018:10:30:04.217]
54.81.204.180 [26/Jan/2018:12:25:35.395]
54.227.52.160 [26/Jan/2018:12:29:20.708]
54.167.233.250 [26/Jan/2018:12:43:36.524]
54.81.197.243 [26/Jan/2018:12:58:15.789]
54.226.73.62 [26/Jan/2018:13:04:54.533]
54.227.52.160 [26/Jan/2018:13:07:51.844]
```

We have a `FIXIE_URL` env var that is being used here:
* https://github.com/travis-ci/travis-tasks/blob/master/lib/travis/tasks/config.rb#L30
* https://github.com/travis-ci/travis-tasks/blob/master/lib/travis/task.rb#L102
* https://github.com/travis-ci/travis-tasks/blob/master/lib/travis/task.rb#L93

Making a Fixie request with `curl` locally (by setting `HTTPS_PROXY=$(heroku config:get FIXIE_URL -a travis-pro-tasks-production)` produces a proper source IP in requestb.in, which suggests it's not a problem with Fixie, but rather more likely in our tasks code.

The dependencies of tasks have been upgraded recently, so that might be another avenue.

(Thanks @vtorhonen for the report and @igorwwwwwwwwwwwwwwwwwwww for info above)

**Comment by soulshake** [comment]
It looks like our Faraday version was upgraded in this [gem upgrade commit](https://github.com/travis-ci/travis-tasks/commit/16a1e295c276e1c206f627fedb090accdba92ac2).

[This issue](https://github.com/lostisland/faraday/issues/723) ("Manually setting a proxy without using the environment") appeared in Faraday after 0.13.0.

We're now using Faraday v0.13.1, and [there's a fix in v0.14.0](https://github.com/lostisland/faraday/commit/4d1ddc130bd224adf25a19eaf0e81fd236a47285).

**Comment by soulshake** [comment]
Fix is in production on com and org :white_check_mark:
